src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -296,10 +304,9 @@ var ReactDOMFiberComponent = {
   ): Element {
     // We create tags in the namespace of their parent container, except HTML
     // tags get no namespace.
-    var ownerDocument: Document = rootContainerElement.nodeType ===
-      DOCUMENT_NODE
-      ? (rootContainerElement: any)
-      : rootContainerElement.ownerDocument;
+    var ownerDocument: Document = getOwnerDocumentFromRootContainer(
```

Comment:
Can you put the annotation on the function getOwnerDocumentFromRootContainer's definition? It's necessary to "cancel out" the `: any` cast.

Reply:
would I still need this `Document` type? I think flow should be able to infer, right?

Reply:
Right, we don't need this one any more.